package.json

```diff
@@ -15,9 +15,9 @@
     "lint": "eslint --fix --ext .js,.ts packages/*/src/**/*",
     "lint:check": "eslint --ext .js,.ts packages/*/src/**/*",
     "test": "npm run test:node && npm run test:browser && npm run test:docs",
-    "test:threads:node": "npm run test:node -- -g Client",
+    "test:threads:node": "npm run test:node -- -g \"^Thread\"",
```

Comment:
Renamed all threads based tests to Start with Thread... either ThreadDB or Thread Client.